django_ses_plus/utils.py

```diff
@@ -1,6 +1,25 @@
+import base64
 from time import strftime
 
+from django.core.mail import get_connection
+from django.core.mail.message import EmailMultiAlternatives
+
 
 def sent_email_upload_path(sent_email, filename):
     filename = filename.split(".")[0]
     return strftime("emails/%Y/%m/%d/%Y-%m-%d-%H-%M-%S-{}.html".format(filename))
+
+
+def send_mail(subject, message, from_email, recipient_list, attachments=None, fail_silently=False, html_message=None):
+    connection = get_connection(fail_silently=fail_silently)
+
+    mail = EmailMultiAlternatives(subject, message, from_email, recipient_list, connection=connection)
+
+    if html_message:
+        mail.attach_alternative(html_message, 'text/html')
+
+    if attachments:
+        for attachment in attachments:
+            mail.attach(attachment["name"], base64.b64decode(attachment["content"]), attachment["type"])
+
+    return mail.send()
```

Comment:
This is the simplified implementation of `django.core.email.send_mail` with `attachments`.

Reply:
Is there any chance to return mail object too? After calling send, ses backend adds, `message_id` to mail object (`mail.extra_headers['message_id']`) add I will need to add this message id to `SentEmail` object for tracking. I can do this change in the upcoming PR but, discussing it now can be beneficial I think.

```
mail = generate_mail()
mail.send()

SentEmail.objects.create(
    ...
    mail.extra_headers['message_id']
    ...
)
```

or 

```
num_sent, mail = send_mail(...)

SentEmail.objects.create(
    ...
    mail.extra_headers['message_id']
    ...
)
```

Reply:
It makes sense to me. I will return mail object instead of returning `mail.send_mail(...)`.

Reply:
thank you!

Reply:
You are welcome! 🚀